### PR TITLE
function addCustomControlMappings()

### PR DESCRIPTION
**Pull request by Lynesth**
Just to allow doing
```php
$menu->addCustomControlMappings([
    "a" => $thisCallable,
    "b" => $otherCallable,
    "c" => $someCallable,
    "x" => new ExitAction
]);
```

instead of
```php
$menu->addCustomControlMapping("a", $thisCallable);
$menu->addCustomControlMapping("b", $otherCallable);
$menu->addCustomControlMapping("c", $someCallable);
$menu->addCustomControlMapping("x", new ExitAction);
```

### CODE REVIEW

**Comment by Lynesth** [comment]
I finally worked out this whole phpunit thing. I'm just not undertanding why the last test fails.
I'm counting on your help @AydinHassan :p

File: test/CliMenuTest.php [hunk 1]
```diff
@@ -480,6 +480,66 @@ public function testAddCustomControlMapping() : void
...
+            }
+        ]);
+    }
+
+    public function testAddCustomControlMappings() : void
+    {
+        $first = true;
+        $this->terminal->expects($this->any())
+            ->method('read')
+            ->will(
```
Comment (AydinHassan) [inline test/CliMenuTest.php]: Here you can just use `->willReturn('c', 'x')`. `c` will be returned on the first call to `read` and `x` on the next :)

File: test/CliMenuTest.php [hunk 2]
```diff
@@ -480,6 +480,66 @@ public function testAddCustomControlMapping() : void
...
+        ]);
+    }
+
+    public function testAddCustomControlMappingsThrowsExceptionWhenAttemptingToOverwriteAddedCustomControlMap() : void
+    {
+        $this->expectException(\InvalidArgumentException::class);
+        $this->expectExceptionMessage('Cannot rebind this input');
+
+        $menu = new CliMenu('PHP School FTW', []);
+        $menu->addCustomControlMappings([
```
Comment (AydinHassan) [inline test/CliMenuTest.php]: It took me a little while to figure this - you actually only send one element to the method, the second `c` key overwrites the first ! So it's actually not possible to call this method once and cause a rebind exception, you'd have to call it twice.

Comment (Lynesth) [reply test/CliMenuTest.php]: Hahaha, of course ! Thank you !

File: test/CliMenuTest.php (hunk 1 above)
Comment (Lynesth) [reply test/CliMenuTest.php]: No ! You've got to be kinding me. What will it return on subsequent calls ? 'x' ?

Comment (AydinHassan) [reply test/CliMenuTest.php]: I think it just returns null after anything you don't specify.

**Pull request merged**